templates/zerver/help/change-your-language.md

```diff
@@ -1,37 +1,42 @@
 # Change your language
 
-By default, the Zulip UI is displayed in English.  However, Zulip has
-full support for internationalization, and the UI has been translated
-into many other languages.  You can see the Zulip UI in other
-languages temporarily by modifying the URLs using the appropriate
-language code, e.g. `/ru/` for Russian.
+By default, the Zulip UI is displayed in English.  However, Zulip has full
+support for internationalization. You can see which languages Zulip supports,
+and help add support for new languages by visiting
+**[Transifex](https://www.transifex.com/zulip/zulip/)**.
 
-Additionally, you can change the default language in the Zulip UI for
-your Zulip account using the following procedure.
-
-!!! warn ""
-    **Warning**: Because for languages, only some of the strings in
-    the Zulip UI have been translated, you may still see English words
-    interspersed throughout Zulip.
+You can change your personal default language for Zulip using the following
+procedure.
 
 {settings_tab|display-settings}
 
-2. Next to the **Default language** heading, select the **[Change]** link.
+2. Under the **Language Settings** heading, select the language next to the
+**Default language** label.
 
-3. After clicking the **[Change]** link, a modal window will appear with all
-foreign languages that Zulip supports along with their percentages of
-translation support. Select your desired language from the window.
+3. After clicking the language link, a modal window will appear with all
+the languages that Zulip supports along with their percentages of
+translation support. Select your desired language.
 
 4. Once you select a language, the modal window will close. Zulip will be
 displayed in the language of your choice after you reload the page.
 
-!!! tip ""
-    You can help translate parts of Zulip by signing up for an account
-    at **[Transifex](https://www.transifex.com/zulip/zulip/)**.
+!!! warn ""
+    **Warning**: For languages that have been less than 100% translated,
+    you may still see some English throughout Zulip.
 
 !!! tip ""
-
     If your entire organization speaks a language other than English, an administrator can
     [set the default language for new users joining the organization][change-org-lang]
 
 [change-org-lang]: change-the-default-language-for-your-organization
+
+## Temporarily see the Zulip UI in other languages
```

Comment:
Nice idea to separate this out. I decided in the end this wasn't actually a useful end user feature (it belongs more in developer documentation), so removed it. (Possibly we should double check that it is indeed in the developer documentation for translating. Would be in readthedocs.)